From 45e6fe8c3978bf3da793d2231bb9d2ac3e2954aa Mon Sep 17 00:00:00 2001
From: Serguei Sagalovitch <Serguei.Sagalovitch@amd.com>
Date: Thu, 24 Mar 2016 11:29:02 -0400
Subject: [PATCH 1758/2094] drm/amdkfd: Turn pr_err to pr_debug call v2

The scenario when we are not able to find process for address is
completely normal when we run non-HSA applications. Switch to
pr_debug to see message only when we want to debug.

v2: Fixed spelling in the commit message

Change-Id: Ia84055dd28c9ebda3dc427ecad860f34776e1835
Signed-off-by: Serguei Sagalovitch <Serguei.Sagalovitch@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_rdma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c b/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c
index 69bdaf1..f58b57e 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_rdma.c
@@ -225,7 +225,7 @@ static int is_gpu_address(uint64_t address, struct pid *pid)
 
 	p = kfd_lookup_process_by_pid(pid);
 	if (!p) {
-		pr_err("could not find the process in %s.\n",
+		pr_debug("could not find the process in %s.\n",
 				__func__);
 		return 0;
 	}
-- 
2.7.4

